tf2_ros/include/tf2_ros/buffer_interface.h

```diff
@@ -55,11 +55,10 @@ namespace tf2_ros
     return time_msg;
   }
 
-  inline const tf2::TimePoint& fromMsg(const builtin_interfaces::msg::Time & time_msg)
+  inline const tf2::TimePoint fromMsg(const builtin_interfaces::msg::Time & time_msg)
```

Comment:
Hm, good point.  I'm guessing we actually *don't* want it to be const return, since it is a translation, and users may want to manipulate the returned object later.  Does that make sense?  If so, I'll remove the const.